### ISSUE TITLE
ProQuest MLA International: Can we get to the RIS?

### ISSUE THREAD

**Issue by adam3smith** [zotero/translators#1483]
Example: https://search.proquest.com/mlaib/docview/742289692/

While we can improve scraping from the page, it's never going to be as good as the RIS. Unfortunately, RIS appears quite hard to get -- can we get to it at all? If so, might be preferable across PQ.
I wasn't successful in initial attempts.

**Comment by zuphilip** [comment]
Is this a general issue with ProQuest databases? I don't have access through ProQuest to the MLA International Bibliography.

![mla-proquest](https://user-images.githubusercontent.com/5199995/33239847-4b41a64c-d2ab-11e7-873a-27acc1a41626.png)

I have to access this database differently (through redi-bw.de) and then I land on the usual EBSCOhost:

![mla-ebsco](https://user-images.githubusercontent.com/5199995/33239806-86cb3508-d2aa-11e7-8fd3-c6db5655c0b9.png)

Is this special for this database or do all ProQuest database look the same for you? If yes, we could try to look in general for RIS data in ProQuest database...

**Comment by adam3smith** [comment]
Yes, exactly the same as in any other PQ database, i.e. click on "Save," Select "RIS" and then "Continue". The problem specific to MLA IB is that the scraping we use on other PQ pages doesn't work well and I'm not seeing a reasonable way to make it -- but the RIS access could be used anywhere on PQ.

**Comment by zuphilip** [comment]
Okay, I only briefly looked at it. The export for BibTeX which is under `Cite` looks for me simplier to simulate. I see two main calls to some sort of API:
* The first (POST) call seem to set the output format: https://search.proquest.com/docview.docviewtoolssection_0.citethis:styleclick/BIB_TEX
* The second (POST) call is then giving the BibTeX data as part of a larger result: https://search.proquest.com/docview.docviewtoolssection_0.citethis.updatecitationform/DocView/$N/citeThisZonecitethis/MSTAR_231115706

It might be important to send all parameter values, but they should be available from the page itself.
